src/pages/index.js

```diff
@@ -62,15 +31,6 @@ api
       const cardElement = getCardElement(item);
       cardsList.append(cardElement);
     });
-
```

Comment:
```js

    document.querySelector(".profile__avatar").src = userInfo.avatar;

```

You've already found `profileAvatar`, so you need to reuse it in line 25 for the avatar setting


```js

    profileAvatar.src = userInfo.avatar;

```